helm-deploy-test/tasks/run-test.sh

```diff
@@ -1,16 +1,15 @@
 #!/bin/bash
 set -o errexit -o nounset
 
-# Export kube-host details from pool
+# Set kube config from pool
+mkdir /root/.kube/ && cp  pool.kube-hosts/metadata /root/.kube/config
+
 set -o allexport
-source pool.kube-hosts/metadata
-CF_NAMESPACE=cf
+DOMAIN=$(ruby -r yaml -e "puts YAML.load_file('pool.kube-hosts/metadata')['contexts'][0]['context']['cluster']").nip.io
```

Comment:
Hmm. Tried to find something else, but the result actually looks worse:
```bash
kubectl config view -o jsonpath={.clusters[0].cluster.server} | ruby -ruri -pe '$_=URI.parse($_).host'
```
Ah, your code assumes the context name is correct, which seems… fishy.

Reply:
Something like this maybe?
```bash
kubectl get nodes --output=jsonpath='{.items[].status.addresses[?(@.type=="Hostname")].address}' | head -n1
```

Reply:
done. But did not retest the whole pipeline

Reply:
looks okay: https://ci.from-the.cloud/teams/main/pipelines/prabal-helm-deploy/jobs/helm-deploy-test/builds/6?